app/assets/javascripts/admins/app.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { RouteHandler, Router, Route, IndexRoute, browserHistory } from 'react-router'
+import { render } from 'react-dom'
+
+import AdminDashboard from './components/AdminDashboard'
+
+render((
+  <Router history={browserHistory}>
+    <Route path='/dashboard' component={AdminDashboard} />
```

Comment:
I like wrapping my strings in `{}` for consistency - though it makes no difference.